### DISCUSSION TITLE
New ver might never come out.

### DISCUSSION THREAD

**Discussion by ethkoth** [3kh0/ext-remover#267] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Echo isn't the dev of the script, he just uploads it to github and the original dev made a post here https://github.com/FogNetwork/Ingot/issues/13 basically saying that he won't be able to fix the exploit for users on 106 or above. 

):

**Comment by ehunt34** [comment]
Exactly what I just said.

**Comment by 3kh0** [comment, marked as answer]
No shit sherlock, i never said i was the dev

**Reply by ethkoth** [reply]
calm down dude. I'm just telling people the update might not release.

**Comment by Theblackmamba824** [comment]
> Echo isn't the dev of the script, he just uploads it to github and the original dev made a post here [FogNetwork/Ingot#13](https://github.com/FogNetwork/Ingot/issues/13) basically saying that he won't be able to fix the exploit for users on 106 or above.
> 
> ):

so they have to find a whole new exploit that doesn't involve ingot

**Reply by Idunnoman001** [reply]
basically.

**Comment by ItzJoseD** [comment]
We're using this to get internet freedom right? I know another way to unblock websites, its a few ehh, "deployments" i made (Really just exported and changed) If you would like i can send a few of my premium web proxies (Best part- it doesnt save to your history and lightspeed+ Gopher will only see url of web proxy, not what your doing or looking at!)

**Reply by Theblackmamba824** [reply]
> We're using this to get internet freedom right? I know another way to unblock websites, its a few ehh, "deployments" i made (Really just exported and changed) If you would like i can send a few of my premium web proxies (Best part- it doesnt save to your history and lightspeed+ Gopher will only see url of web proxy, not what your doing or looking at!)

pls send

**Comment by Aka-but-nice** [comment]
Why ya'll acting like Avengers I got something that should unblock stuff. Stop acting like boss babies and grow up but here anyways.
Here's what you have to do. Make a bookmark with chrome://restart. Next, click on it and your Chromebook should restart, AS SOON AS IT COMES BACK UP CLICK THE CALENDER AT THE BOTTOM OF THE SCREEN and it will restart again then everything will be unblocked if you don't have a calendar at the bottom of the screen, go to chrome://flags and look up a calendar and enable anything that's there. It works for me if it doesn't work for you, well that's sad.
